src/mmw/apps/modeling/migrations/0008_scenario_results_to_json.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def convert_to_json(apps, schema_editor):
```

Comment:
Initially I had attempted to convert existing results to proper JSON, but clearing them out was simpler. I should probably update this functions name to reflect that.